### PR TITLE
Xstate: second login and stay connected

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -118,6 +135,7 @@ class Masq {
           }
         },
         logged: {
+          entry: [ _storeSessionInfo ],
           on: {
             SIGNOUT: {
               target: 'notLogged',
```
Comment (G-Ray) [inline src/index.js]: every target should be prefixed with `#Masq.target` ?

Comment (sujeebant) [reply src/index.js]: #Masq prefix is necessary only when indicating a target located in the 'Masq' machine from another machine.

File: src/index.js [hunk 2]
```diff
@@ -509,6 +585,7 @@ const _registering = (context, event) => async (cbParent) => {
 }
 
 const _readSessionInfoIntoState = async (context, event) => {
+  console.log('[_readSessionInfoIntoState]')
```
Comment (G-Ray) [inline src/index.js]: debug ?

Comment (sujeebant) [reply src/index.js]: will be removed in a future PR to xstate branch :+1:

**Pull request merged**